retroachievements/client.py

```diff
@@ -0,0 +1,127 @@
+import requests as request
+
+
+class RAClient:
+
+    # Header (required to pass cloudflare consistantly)
+    headers = {
+        "User-Agent": "api-python/0.1 - RetroAchievements Web API Python Wrapper"
```

Comment:
"User-Agent": "retroachievements-api-python/0.1 - RetroAchievements Web API Python Client"

@wescopeland we should probably standardize this across clients as well?

Reply:
We should indeed. I propose:

```
RetroAchievements-api-js/1.X.X (any other useful metadata here)
```

If this is agreeable, I will get it into api-js right away.

Reply:
I'd aim for a 1.0 release right away

Reply:
👍 Tracking in api-js at https://github.com/RetroAchievements/api-js/issues/75